processor/build.gradle

```diff
@@ -36,8 +36,8 @@ dependencies {
     testCompile "commons-io:commons-io:$COMMONS_IO_VERSION"
     testCompile androidJar()
     testCompile files(Jvm.current().getToolsJar())
-    testCompileAar "com.android.support:support-v4:$SUPPORT_LIBRARY_VERSION"
-    testCompileAar "com.android.support:support-v13:$SUPPORT_LIBRARY_VERSION"
+    testCompileAar 'androidx.legacy:legacy-support-v4:1.0.0-alpha3'
+    testCompileAar 'androidx.legacy:legacy-support-v13:1.0.0-alpha3'
```

Comment:
Create a version code at `gradle.properties`

Reply:
TO: @hotchemi @mannodermaus 

I was thinking we are still keep android support library's support. Do we want to drop the support of support library?